.github/actions/e2e_verify/action.yml

```diff
@@ -66,36 +66,32 @@ runs:
           forwarderPID=$!
           sleep 5
 
-          verifyOut=$(constellation verify --cluster-id "${clusterID}" --force --node-endpoint localhost:9090)
-
-          kill $forwarderPID
-
-          if [[ ${{ inputs.cloudProvider }} != "azure" ]]; then
-            continue
+          if [[ ${{ inputs.cloudProvider }} == "azure" ]]; then
+            echo "Extracting Azure TCB versions for API update"
+            constellation verify --cluster-id "${clusterID}" --force --node-endpoint localhost:9090 -o json > "snp-report-${node}.json"
+          else
+            constellation verify --cluster-id "${clusterID}" --force --node-endpoint localhost:9090
           fi
 
-          echo "Extracting TCB versions for API update"
-          startMAAToken="Microsoft Azure Attestation Token:"
-          endMAAToken="Verification OK"
-          sed -n "/${startMAAToken}/,/${endMAAToken}/ { /${startMAAToken}/d; /${endMAAToken}/d; p }" <<< "${verifyOut}" > "maa-claims-${node}.json"
+          kill $forwarderPID
         done
 
     - name: Login to AWS
-      if: github.ref_name == 'main' && inputs.cloudProvider == 'azure'
+      if: inputs.cloudProvider == 'azure'
```

Comment:
I would like to keep the check: unreviewed code shouldn't push to our production API.

Since this only affects the upload step the action can still be run on the feature branch. This will catch syntax errors. The CLI itself can be tested with the e2e-attestationconfigapi test. Incorrect invocations will not be caught, which I agree is an inconvenience. But in this case it's better to live with this small inconvenience over pushing malformed data to the API imo.

Reply:
I see the potential danger, but I still believe that we should disable this check temporarily after review and test before merging into main.

Reply:
I removed the check again